src/pudl/metadata/fields.py

```diff
@@ -466,6 +466,18 @@
         "type": "number",
         "description": "Fraction of potential generation that was actually reported for a plant part.",
     },
+    "capacity_factor_offshore_wind": {
+        "type": "number",
+        "description": "Fraction of potential generation calculated for offshore wind.",
+    },
+    "capacity_factor_onshore_wind": {
+        "type": "number",
+        "description": "Fraction of potential generation calculated for onshore wind.",
+    },
+    "capacity_factor_solar_pv": {
+        "type": "number",
+        "description": "Fraction of potential generation calculated for solar pv.",
```

Comment:
```suggestion
        "description": "Fraction of potential generation calculated for fixed-tilt solar PV installations.",
```

Reply:
Might also specify that the tilt is equal to the latitude.